tests/unit/ui/components/TestShowMoreCard.js

```diff
@@ -69,4 +69,23 @@ describe('<ShowMoreCard />', () => {
 
     sinon.assert.calledWith(truncateToMaxHeight, contentNode);
   });
+
+  it('expands if new child content is smaller', () => {
+    const root = mount(
+      <ShowMoreCardBase i18n={getFakeI18nInst()}>
+        This should be a very long content, but we cannot get `clientHeight` in
```

Comment:
Nitpick but this would be better written as "This would be very long content, but [...]"